### PR TITLE
Improve "How to test" sections of criteria

### LINKED ISSUE #314: Not all MUST criteria have a clear test in the "How to test" section
As mentioned in Issue https://github.com/publiccodenet/standard/issues/80 and Issue https://github.com/publiccodenet/standard/issues/141, the section 'How to test' has several problems:

1. They don't cover all the MUST
2. Not all of them are clear tests, but more like key evaluation concepts

Can we ensure that each criterion has a clear test and make it similarly clear which requirement each test covers?

### PR DESCRIPTION
**Pull request by ericherman**
fixes #314

-----
[View rendered criteria/bundle-policy-and-code.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/bundle-policy-and-code.md)
[View rendered criteria/code-in-the-open.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/code-in-the-open.md)
[View rendered criteria/continuous-integration.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/continuous-integration.md)
[View rendered criteria/document-maturity.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/document-maturity.md)
[View rendered criteria/document-objectives.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/document-objectives.md)
[View rendered criteria/documenting.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/documenting.md)
[View rendered criteria/make-contributing-easy.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/make-contributing-easy.md)
[View rendered criteria/open-licenses.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/open-licenses.md)
[View rendered criteria/open-standards.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/open-standards.md)
[View rendered criteria/open-to-contributions.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/open-to-contributions.md)
[View rendered criteria/require-review.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/require-review.md)
[View rendered criteria/reusable-and-portable-codebases.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/reusable-and-portable-codebases.md)
[View rendered criteria/style.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/style.md)
[View rendered criteria/understandable-english-first.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/understandable-english-first.md)
[View rendered criteria/version-control-and-history.md](https://github.com/publiccodenet/standard/blob/improve-how-to-test-314/criteria/version-control-and-history.md)

### CODE REVIEW

File: criteria/document-maturity.md [hunk 1]
```diff
@@ -25,9 +25,11 @@ Clearly signalling a codebase's maturity helps others decide whether to reuse, i
 
 ## How to test
 
-* The codebase has a strategy for versioning which is documented.
-* It is clear where to get the newest version.
-* The codebase doesn't depend on any codebases marked with a less mature status.
+* Confirm that the codebase has a strategy for versioning which is documented.
+* Confirm that is clear where to get the newest version.
```
Comment (Ainali) [inline criteria/document-maturity.md:29]:
```suggestion
* Confirm that it is clear where to get the newest version.
```

File: criteria/make-contributing-easy.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ order: 5
 
 ## How to test
 
-* There's a public issue tracker.
-* It's possible to participate in a discussion with other users about the software.
+* Confirm that there is a public issue tracker.
+* Confirm that there are instructions for privately reporting security issues.
+* Confirm the codebase contains links to the public issue tracker and submitted codebase changes.
```
Comment (Ainali) [inline criteria/make-contributing-easy.md:31]:
```suggestion
* Confirm that the codebase contains links to the public issue tracker and submitted codebase changes.
```

File: criteria/make-contributing-easy.md [hunk 3]
```diff
@@ -26,8 +26,10 @@ order: 5
...
+* Confirm that it is possible to participate in a discussion with other users about the software using channels described in the codebase.
```
Comment (Ainali) [inline criteria/make-contributing-easy.md:32]:
```suggestion
* Confirm that it is possible to participate in a discussion with other users and developers about the software using channels described in the codebase.
```

File: criteria/open-standards.md [hunk 4]
```diff
@@ -27,8 +27,9 @@ order: 11
 
 ## How to test
 
-* The documentation includes a list of the standards followed within the codebase, each with a working link, or a statement that no standards were chosen.
-* The standards used for all features that offer interoperability with other components and systems are freely and publicly available on the internet.
+* Confirm that data exchange follows an OSI approved open standard.
+* Confirm that any non-open standards used are clearly documentation as such.
```
Comment (Ainali) [inline criteria/open-standards.md:31]:
```suggestion
* Confirm that any non-open standards used are clearly documented as such.
```

File: criteria/open-to-contributions.md [hunk 5]
```diff
@@ -29,9 +29,13 @@ order: 4
 
 ## How to test
 
-* It's possible to submit suggestions for changes to the codebase.
-* There are contribution guidelines.
-* Codebase governance is clearly explained, including how to influence codebase governance.
+* Confirm that it is possible to submit suggestions for changes to the codebase.
+* Confirm there are contribution guidelines.
+* Confirm that the codebase governance is clearly explained, including how to influence codebase governance.
+* Check for list of involved organizations.
```
Comment (Ainali) [inline criteria/open-to-contributions.md:35]:
```suggestion
* Check for a list of involved organizations.
```

File: criteria/require-review.md [hunk 6]
```diff
@@ -36,7 +36,14 @@ order: 7
 
 ## How to test
 
-* Every commit in the history has been reviewed by a different contributor in a different context.
+* Confirm that every commit in the history has been reviewed by a different contributor.
+* Confirm that reviews include source, policy, tests and documentation.
+* Confirm that rejected contributions were appropriately explained.
+* Check if reviews cover conformance to standards, architecture and codebase guidelines.
+* Check with reviews if they run the code and tests during review.
```
Comment (Ainali) [inline criteria/require-review.md:43]:
```suggestion
* Check with reviewers if they run the code and tests during review.
```

**Review by Ainali — CHANGES_REQUESTED** [review]
Some tiny wordsmithing needed, but in general, this is great!

File: criteria/documenting.md [hunk 7]
```diff
@@ -31,9 +31,14 @@ order: 9
...
-* Documentation is generated from code.
-* Links and images are automatically tested.
+* Confirm that other stakeholders, professionals from other public organizations and the general public find the documentation clear and understandable.
+* Confirm that the documentation describes how to install and run the source code.
+* Confirm that the documentation includes examples of the key functionality.
+* Check with members of the general public and journalists if they can understand the high level description.
+* Check that the instructions for how to install and run a standalone version of the source code result in a running system.
+* Check that all functionality documented contains an example.
+* Check that the documentation includes a high level architectural diagram or similar.
+* Check that the documentation quality is part of integration testing, for instance documentation is generated from code and links and images are tested.
```
Comment (ElenaFdR) [inline criteria/documenting.md:41]: More colloquial; might be less familiar to a non-native English speaker.
```suggestion
* Check that the documentation quality is part of integration testing, for example documentation is generated from code and links and images are tested.
```

File: criteria/open-licenses.md [hunk 8]
```diff
@@ -25,9 +25,11 @@ order: 13
 
 ## How to test
 
-* There is at least 1 license file present in the codebase, usually called `license`.
-* The license for the source code is on the [OSI-approved Open Source license list](https://opensource.org/licenses/category) and the license for documentation is [conformant to the Open Definition](https://opendefinition.org/licenses/).
-* Add machine-readable license checking to the codebase continuous integration tests.
+* Confirm that the codebase is clearly licensed.
+* Confirm that the license for the source code is on the [OSI-approved Open Source license list](https://opensource.org/licenses/category) and the license for documentation is [conformant to the Open Definition](https://opendefinition.org/licenses/).
+* Confirm that the licenses used in the codebase are included as files.
+* Confirm that contribution guidelines and repository configuration does not require transfer of copyright.
```
Comment (ElenaFdR) [inline criteria/open-licenses.md:31]:
```suggestion
* Confirm that contribution guidelines and repository configuration do not require transfer of copyright.
```

File: criteria/open-standards.md [hunk 9]
```diff
@@ -27,8 +27,9 @@ order: 11
 
 ## How to test
 
-* The documentation includes a list of the standards followed within the codebase, each with a working link, or a statement that no standards were chosen.
-* The standards used for all features that offer interoperability with other components and systems are freely and publicly available on the internet.
+* Confirm that data exchange follows an OSI approved open standard.
```
Comment (ElenaFdR) [inline criteria/open-standards.md:30]: Add hyperlink to OSI-approved list.

**Review by Ainali — APPROVED** [review]

**Pull request merged**